### DISCUSSION TITLE
Print the full path of a menubutton selection

### DISCUSSION THREAD

**Discussion by ruanformigoni** [fltk-rs/fltk-rs#1319] [Q&A] [answered]
Hi, is there a way to print the full path in the selection demonstrated below?

![screenshot-59](https://user-images.githubusercontent.com/18180693/204051295-8acd4aa3-b0e7-42b7-bf1e-366531f8d706.png)

In this case `/home/test/test2`.

**Comment by MoAlyousef** [comment, marked as answer]
Hello

I've added a binding to item_pathname in the master branch.

```rust
some_menu.set_callback(|m| {
    println!("{}", m.item_pathname(None).unwrap());
});
```

**Reply by ruanformigoni** [reply]
Thanks a lot! Do you have a link for donations?

**Reply by MoAlyousef** [reply]
You're welcome. I don't have a link for donation. You could probably donate to FLTK, since most of the work happens there, in addition to the costs of the server and domain. I can't find a link for donation, but if I do, I'll post it here and link to it in the README.
